astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1825,6 +1846,9 @@ def __init__(
         self.is_dataclass: bool = False
         """Whether this class is a dataclass."""
 
+        self.type_params: list[nodes.TypeVar] = []
```

Comment:
This should be expanded to also cover the other types right?

Reply:
Ouch, thanks!